testing/src/cypress/custom-command/common/authorization.ts

```diff
@@ -0,0 +1,27 @@
+import axios, { AxiosResponse } from "axios";
+import { stringify } from "querystring";
+import authData from "../../e2e/data/authentication-details.json";
+
+export default class Authorization {
+  async getAuthToken(): Promise<AxiosResponse<string>> {
```

Comment:
Please check the method return type, I believe that it should be `Promise<string>`.

Reply:
updated